dependencies/windows/install-dependencies.cmd

```diff
@@ -2,14 +2,14 @@
 
 setlocal
 
-set PATH=%PATH%;%CD%\tools
+set PATH=%CD%\tools;%PATH%
 
-set WGET_ARGS=--no-check-certificate
+set WGET_ARGS=-c --no-check-certificate
 set UNZIP_ARGS=-q
```

Comment:
Yes, the world of windows batch scripts is a hall of mirrors! Be warned
before you enter....

On Thu, Jul 7, 2016 at 1:55 AM, Kevin Ushey <notifications@github.com>
wrote:

> In dependencies/windows/install-dependencies.cmd
> <https://github.com/rstudio/rstudio/pull/724#discussion_r69854510>:
>
> >
> > -set WGET_ARGS=--no-check-certificate
> > +set WGET_ARGS=-c --no-check-certificate
> >  set UNZIP_ARGS=-q
>
> I was surprised to discover that attempting to call unzip as e.g.
>
> set UNZIP=unzip -q
> %UNZIP% ...
>
> fails; whereas the current form:
>
> set UNZIP_ARGS=-q
> unzip %UNZIP_ARGS%
>
> works fine. (wget seems to handle both invocations correctly.) Such is
> life in the world of Windows batch scripts, I suppose?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/rstudio/pull/724/files/a221943a042b90a47055de6ec4443af79b9cfeb5#r69854510>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGXx-OAqEP49ZiK4bKUhFrr9kHYBTGEks5qTJTlgaJpZM4JDj5r>
> .
>